index.html

```diff
@@ -0,0 +1,357 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/9ac36e1f47.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="style.css">
+    <title>Youtube</title>
+</head>
+<body>
+    <div class="nav">
+        <div class="logo"><img src="media/YouTube-Logo.wine.svg" alt="logo" width="100px"></div>
+        <div class="search-placeholder">
+           <div class="search">
+            <input type="text" placeholder="Search.." name="search">
+            <button type="submit" class="button-submit"><i class="fa-sharp fa-solid fa-magnifying-glass"></i></button>
+            </div>
+            <span><i class="fa-sharp fa-solid fa-microphone"></i></span>
+        </div>
+
+        <div class="head-icons">
+
+            <div><i class="fa-solid fa-square-plus"></i></div>
+            <div><i class="fa-sharp fa-solid fa-bell"></i></div>
+            <div><i class="fa-solid fa-user"></i></div>
+        </div>
+    </div>
+
+
+
+    <div class="container">
+        <div class="video-box">
+            <video width="100%" height="600" controls>
+                <source src="movie.mp4" type="video/mp4">
+                <source src="movie.ogg" type="video/ogg">
+                Your browser does not support the video tag.
+              </video>
+       
+
+        <div class="about-video">
+            <div class="video-title"><p>Contributing to Open Source Part I: The Easy Way</p></div>
+            <div class="controls">
+                <div class="chanel">
+                    <i class="fa-solid fa-user"></i>
+                    <p>The Odin Project <br> <span>1.11K subscribers</span></p>
+                    <button type="submit" class="sub">Subscribe</button>
+                </div>
+                
+                
+              <div class="icons">
+                <div class="like-dislike">
+                    <button type="submit" class="like"><i class="fa-solid fa-thumbs-up"></i><span>488</span></button>
+                    <button type="submit" class="dislike"><i class="fa-solid fa-thumbs-down"></i></button>
+                </div>
+                <button type="submit" class="share"><i class="fa-solid fa-share"></i><span>Share</span></button>
+                <button type="submit" class="clip"><i class="fa-solid fa-scissors"></i><span>Clip</span></button>
+                <button type="submit" class="save"><i class="fa-solid fa-floppy-disk"></i><span>Save</span></button>
+                <button type="submit" class="more"><i class="fa-solid fa-ellipsis"></i></button>
+            </div>
```

Comment:
Please Use recommended HTML semantic tags...It will help you site to rank well in the search engine .